slo/promql_test.go

```diff
@@ -222,6 +222,33 @@ var (
 			},
 		}
 	}
+	objectiveAPIServerRatioAlertingDisabled = func() Objective {
+		return Objective{
+			Labels:   labels.FromStrings(labels.MetricName, "apiserver-write-response-errors"),
+			Target:   99,
+			Alerting: Alerting{Disabled: true},
+			Window:   model.Duration(14 * 24 * time.Hour),
+			Indicator: Indicator{
+				Ratio: &RatioIndicator{
+					Errors: Metric{
+						Name: "apiserver_request_total",
+						LabelMatchers: []*labels.Matcher{
+							{Type: labels.MatchEqual, Name: "job", Value: "apiserver"},
+							{Type: labels.MatchRegexp, Name: "verb", Value: "POST|PUT|PATCH|DELETE"},
+							{Type: labels.MatchRegexp, Name: "code", Value: "5.."},
+						},
+					},
+					Total: Metric{
+						Name: "apiserver_request_total",
+						LabelMatchers: []*labels.Matcher{
+							{Type: labels.MatchEqual, Name: "job", Value: "apiserver"},
+							{Type: labels.MatchRegexp, Name: "verb", Value: "POST|PUT|PATCH|DELETE"},
+						},
+					},
+				},
+			},
+		}
+	}
```

Comment:
The reason these are functions returning objectives is to allow for creating slight variations. 
This should be the same and more maintainable in the future.
```suggestion
	objectiveAPIServerRatioAlertingDisabled = func() Objective {
		o := objectiveAPIServerRatio()
		o.Alerting.Disabled = true
		return o
	}
```